test/cbu.js

```diff
@@ -21,7 +21,6 @@ describe('CBU', function() {
 
   it("has associated bank", function() {
     cbu.getAssociatedBank('2850590940090418135201').should.be.exactly('Banco Macro S.A.');
-    cbu.getAssociatedBank('0050590940090418135201').should.be.exactly('The Royal Bank of Scotland N.V.');
```

Comment:
I added the cbu verification before the .getAssociatedBank function, and because of this, the test gives me an error because one of the cbu to test is invalid.